CONTRIBUTING.md

```diff
@@ -10,19 +10,31 @@ expectations the Node.js Foundation requires from all contributors.
 
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a contributor who has been given write access to the repository
-* An **Observer** is any individual who has requested or been requested to attend a CommComm meeting. It is also the first step to becoming a Member.
+* An **Observer** is any individual who has requested or been requested to attend a CommComm meeting.
```

Comment:
I know I missed the last couple of meetings, but I seem to recall we had decided to discard the Observer status altogether?

Reply:
I think we agreed on "Participant".

Reply:
I meant we agreed on calling "Participant" not "Observers". I think that was in second last meeting. I missed that last one, so you are right. 👍